File: javascript/maior_posicao.js
const n = parseInt(prompt("Quanto numeros voce vai digitar?"));

let vetor = [], maior, posicaoMaior;

for(i = 0; i < n; i++) {
    vetor[i] = parseFloat(prompt("Digite um numero:"));
}

maior = vetor[0];
posicaoMaior = 0;

for(i = 0; i < n; i++) {
    if (vetor[i] > maior) {
        maior = vetor[i];
        posicaoMaior = i;
    }
}

console.log("MAIOR VALOR = %.1f", maior);
console.log("POSICAO DO MAIOR VALOR = %d", posicaoMaior);

File: python/maior_posicao.py
n = int(input("Quantos numeros voce vai digitar? "))

vetor = []

for i in range(0, n):
    vetor.append(float(input("Digite um numero: ")))

maior = vetor[0]
posicao_maior = 0

for i in range(0, n):
    if vetor[i] > maior:
        maior = vetor[i]
        posicao_maior = i

print()
print(f"MAIOR VALOR = {maior:.1f}")
print("POSICAO DO MAIOR VALOR =", posicao_maior)
